integration-test/src/test/java/org/apache/gravitino/integration/test/trino/TrinoQueryTestTool.java

```diff
@@ -47,6 +47,10 @@ public static void main(String[] args) throws Exception {
           false,
           "Generate the output file for the test set, the default value is 'false'");
 
+      options.addOption(
+          "test_host",
+          true,
+          "host for test server, if --auto is set to 'all', this option is ignored");
```

Comment:
what's the meaning of `test server`?

Reply:
The services server for run the tester. eg: hive, mysql, trino, gravitino server ...

Reply:
`host for test server` is very vague, can you give some examples about the possible value? besides `host for test server` means the the `type` of server or `address` of the server?